src/Components/Book.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function Book() {
+  return (
+    <>
+      <div>
+        title
+      </div>
+      <div>
+        Author
+      </div>
+      <button type="submit"> Delete</button>
+    </>
+  );
+}
```

Comment:
- [ ] Kindly make the `Book` component receive a `title` and `author` as props as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md#:~:text=Create%20a%20component%20called%20Book%20for%20displaying%20a%20single%20book%20(receive%20a%20title%20and%20an%20author%20as%20prop)%20and%20reuse%20it%20in%20a%20component%20that%20displays%20a%20list%20of%20books.).